### PR TITLE
Renovate Coverage Report Generation & Upload

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: src/api/Gemfile [hunk 1]
```diff
@@ -143,8 +143,8 @@ group :test do
   # to ensure a clean state for testing
   gem 'database_cleaner-active_record'
   # for test coverage reports
-  gem 'codecov', require: false
   gem 'simplecov', require: false
```
Comment (eduardoj) [inline src/api/Gemfile:146]:
```suggestion
```
`simplecov` will become a dependency of `simplecov-cobertura`. Do we still need to include it explicitly?

File: .circleci/conditional_config.yml [hunk 2]
```diff
@@ -365,16 +365,32 @@ jobs:
...
             cd src/api
-            export COVERALLS_REPO_TOKEN=HWLJwfiFsKPGEOzfgllO3pP3rqV540Qt3
             bundle exec rake ci:simplecov_ci_merge
+      - run:
+          name: Upload coverage to codecov
+          command: |
+            cd src/api
+            curl -LOs https://uploader.codecov.io/latest/linux/codecov
+            chmod +x codecov
+            ./codecov -v -f coverage/coverage.xml
```
Comment (krauselukas) [inline .circleci/conditional_config.yml:378]: @hennevogel there is not Token required for the codecov upload?

Comment (hennevogel) [reply .circleci/conditional_config.yml:378]: nope

**Review by krauselukas — APPROVED** [review]

**Comment by hennevogel** [comment]
Codecov hangs. Codeclimate is fine though. See the checks. Merging!

**Pull request merged**